docs/docsource/operation/index.rst

```diff
@@ -17,6 +17,7 @@ such as Google Cloud Platform (GCP) or Microsoft Azure.
    About |product|<Introduction/index>
    Architecture<Architecture/index>
    Services<Services/index>
-   Using the |product| UI<GUI/index>
+   Using Orbit<Orbit_UI/index>
+   The |product| UI<XDM_GUI/index>
```

Comment:
Adding `The` here feels out of place to me, it does not seem like something I see in other technical docs.

Reply:
It kind of depends on how you're presenting the product to the customer. If you want to keep it at arm's length from the product, as a free-standing tool, I'd say "ditch the article" and go with "XDM (or whatever) UI," but using the article sounds more natural to my ear.

Reply:
hmm "The Dashboard" feels weird to me too... I am not the expert here so will defer judgment of course.